tests/plots/test_declarative.py

```diff
@@ -1150,6 +1150,66 @@ def test_declarative_multiple_sfc_obs_change_units(ccrs):
     return pc.figure
 
 
+@pytest.mark.mpl_image_compare(remove_text=False, tolerance=0.607)
+@needs_cartopy
+def test_declarative_font_sizes():
+    """Test all font size attributes"""
+    data = xr.open_dataset(get_test_data('GFS_test.nc', as_file_obj=False))
+
+    cntr = ContourPlot()
+    cntr.data = data
+    cntr.field = 'Geopotential_height_isobaric'
+    cntr.level = 300 * units.hPa
+    cntr.time = datetime(2010, 10, 26, 12)
+    cntr.contours = list(range(0, 10000, 120))
+    cntr.linecolor = 'black'
+    cntr.linestyle = 'solid'
+    cntr.clabels = True
+    cntr.clabelsize = 18
+
+    cfill = FilledContourPlot()
+    cfill.data = data
+    cfill.field = 'Temperature_isobaric'
+    cfill.level = 300 * units.hPa
+    cfill.time = datetime(2010, 10, 26, 12)
+    cfill.contours = list(range(210, 250, 2))
+    cfill.colormap = 'BuPu'
+    cfill.colorbar = 'horizontal'
+    cfill.colorbar_fontsize = 'x-small'
+
+    data2 = parse_metar_file(get_test_data('metar_20190701_1200.txt',
+                                           as_file_obj=False), year=2019, month=7)
+    obs = PlotObs()
+    obs.data = data2
+    obs.time = datetime(2019, 7, 1, 12)
+    obs.time_window = timedelta(minutes=15)
+    obs.level = None
+    obs.fields = ['cloud_coverage', 'air_temperature', 'dew_point_temperature',
+                  'air_pressure_at_sea_level', 'present_weather']
```

Comment:
`'present_weather'` should now be `'current_wx1_symbol'` as the result of PR #1876

Reply:
Super helpful, thank you!